data/CMakeLists.txt

```diff
@@ -90,6 +90,33 @@ if (FREECIV_ENABLE_CLIENT OR FREECIV_ENABLE_CIVMANUAL)
     COMPONENT freeciv21)
 endif()
 
+# Icons
+if (FREECIV_ENABLE_CLIENT OR FREECIV_ENABLE_SERVER OR
+    FREECIV_ENABLE_RULEDIT OR FREECIV_ENABLE_FCMP_QT)
+  install(FILES ${CMAKE_SOURCE_DIR}/data/icons/index.theme
+          DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/freeciv21/icons
```

Comment:
DESTINATION "${CMAKE_INSTALL_DATADIR}/${PROJECT_NAME}"/icons

as you have it, it installs to {root}/freeciv21/icons on msys2.

Reply:
in the same file you can see the usage of the DESTINATION directive

Reply:
I hope I got it right this time :sweat_smile: